src/frontend/komponenter/Behandling/VedtakOgBeregning/VedtakOgBeregning.tsx

```diff
@@ -38,13 +38,29 @@ const VedtakOgBeregning: FC<Props> = ({ behandlingId }) => {
         [behandlingId]
     );
 
+    const lagretVedtakConfig: AxiosRequestConfig = useMemo(
+        () => ({
+            method: 'GET',
+            url: `/familie-ef-sak/api/vedtak/${behandlingId}`,
+        }),
+        [behandlingId]
+    );
+
     const søknadDataResponse: Ressurs<ISøknadData> = useDataHenter<ISøknadData, null>(
         søknadDataConfig
     );
 
+    const lagretVedtakResponse: Ressurs<IVedtak | undefined> = useDataHenter<
+        IVedtak | undefined,
+        null
+    >(lagretVedtakConfig);
+
     return (
-        <DataViewer response={{ søknadDataResponse }}>
-            {({ søknadDataResponse }) => {
+        <DataViewer response={{ søknadDataResponse, lagretVedtakResponse }}>
+            {({ søknadDataResponse, lagretVedtakResponse }) => {
+                if (lagretVedtakResponse) {
+                    settResultatType(lagretVedtakResponse.resultatType);
+                }
```

Comment:
Risikrerer vi evig løkke her? 🤔

Reply:
Hvis ja; da må isåfall lagretVedtak pakkes ut i en `useEffect({},[])` og resultatType settes der.

Reply:
Troor ikke evig løkke kan skje, men kan gjøre det vanskelig redigere vedtak hvis finnes.